packages/catalog-process/src/services/catalogService.ts

```diff
@@ -2956,6 +2956,7 @@ export function catalogServiceBuilder(
             templateId: template.id,
             templateVersionId: publishedVersion.id,
             instanceLabel: seed.instanceLabel,
+            templateAttributes: publishedVersion.attributes,
```

Comment:
I think you need to modify also `innerCreateEservice` to actually set these attributes as the descriptor attributes, at the moment they are hardcoded: https://vscode.dev/github/pagopa/interop-be-monorepo/blob/develop/packages/catalog-process/src/services/catalogService.ts#L564

Reply:
Ups